lib/commands/general.js

```diff
@@ -339,6 +339,10 @@ commands.getSystemBars = async function () {
   return parseWindows(out);
 };
 
+commands.broadcastIntent = async function (intent) {
```

Comment:
sometimes this might to be enough to just have intent name, since broadcasting requires additional arguments to be provided

Reply:
We would have to update `appium-adb` to include arguments to with `broadcast` (https://developer.android.com/studio/command-line/adb.html#IntentSpec) which we could easily do but I wonder, do you foresee any security issues?

Reply:
I don't think so. If we provide arguments as a simple array then these are going to be escaped automatically by the underlying `spawn` call.